README.md

```diff
@@ -3,7 +3,7 @@
 hi yes this is the shitcode of bolb bot.
 have fun.
 
-written by [koala#9712](https://github.com/koala9712), [Sham#0002](https://github.com/realShamlol) and [ooliver1](<https://github.com/ooliver> 1) ofc
+written by [koala#9712](https://github.com/koala9712), [Sham#0002](https://github.com/realShamlol) and [ooliver1](<https://github.com/ooliver>) ofc
```

Comment:
You said https://github.com/ooliver again 😂 , please just tell me if it is another account you own, or change it to ooliver1 (the account you are using to commit this).

Reply:
I don't know why I keep doing that